src/backend/marsha/core/models/file.py

```diff
@@ -213,3 +213,21 @@ def update_upload_state(self, upload_state, uploaded_on, **extra_parameters):
         """
         self.extension = extra_parameters.get("extension")
         super().update_upload_state(upload_state, uploaded_on, **extra_parameters)
+
+
+class AbstractImage(UploadableFileMixin, BaseModel):
+    """Abstract model for images."""
+
+    class Meta:
+        """Options for the ``AbstractImage`` model."""
+
+        abstract = True
+
+    @property
+    def is_ready_to_show(self):
```

Comment:
from what I looked, it seems that this method is now shared by all models using the `UploadableFileMixin` so it should probably be defined once on the mixin?

Reply:
Yes you are right, I will make this modification.

Reply:
I made the modification in this commit d8d45bdc6d0c752b3949826682366e2ec5d2603c